src/components/Footer.tsx

```diff
@@ -1,6 +1,6 @@
 const Copyright = () => {
   return (
-    <p className="text-center text-neutral-400 dark:text-neutral-600">
+    <p className="text-center text-neutral-700 dark:text-neutral-400">
```

Comment:
I made the text specifically to be subtle that's why they were not bright. Could you revert your changes to the original. thx

Reply:
I liked the colors of the site and I don't want to change them, but some colors reduce readability.